server/src/models/forest-locations.es6

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const url = require('url');
+
+const sequelizeOptions = {
+  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0]
+};
+
+if (
```

Comment:
why are these being defined on each model? this seems like this should be specified at runtime or in the sequeliz.rc, and not at the individual model level.

Reply:
@lauraGgit we'll research and refactor this.

Reply:
@lpendem could you please let me know which story was created to address this?

Reply:
@lauraGgit fixed this in sorint-2 story https://github.com/nciinc/fs-intake-module/issues/6